### ISSUE TITLE
Addressing Zap-Solutions-Strike concerns

### ISSUE THREAD

**Issue by JimmyMow** [Enegnei/This-Month-In-Bitcoin-Privacy#2]
Hello. This issue is intended to reply to [this](https://github.com/Enegnei/This-Month-In-Bitcoin-Privacy/blob/master/_posts/where-data-flows/Zap-Solutions-Strike.md) post authored by @Enegnei and make any needed changes to our (Zap/Strike) privacy policy, terms, product, and so on.

> The Strike homepage describes their service as 'private.'

I agree this can be misleading. The intention is data we have is never shared and one can spend money to friends and on services through us as a proxy of sorts. However, with Cognito and Plaid, I can understand the concern with displaying `Private` without further clarification. This has been updated to `Fast` on our [website](https://beta.strike.me/).

> They also reserve the right to "block access to the Services from certain IP addresses and unique device identifiers."

Of course. This is not unique to our service. If we need to block service for a user for regulatory, compliance, or malicious behavior we need to ensure we have that right.

> Transaction terms

Reserving the right to change limits is based on our ongoing dialog with regulators. Currently, a LIGHT user has the following limits:

- $100 per payment
- $2,000 lifetime

I work with regulators to raise these limits and eventually remove them. This was [discussed publicly](https://www.youtube.com/watch?v=BKxrLCzMLLo) (on your podcast @Enegnei)

> However they also use "pixel tags / web beacons," and engage in other automated collection activities:

Technology such as Cookies is required for such features as persisting a user's login. Web beacons or pixel tags are the politically correct, lawyer-speak, way of acknowledging we use analytical tools. The definition is defined in the Privacy Policy. This is standard across all web and mobile applications.

> Cognito

Yes, it is true that we use Cognito to do a wider search based on the information provided to us. [I clarified publicly](https://www.youtube.com/watch?v=BKxrLCzMLLo) (on your podcast @Enegnei) that this "LIGHT KYC" effort was designed to prevent users from sharing information with a third party (Strike) if they don't have to. One of the many dangers KYC authors is a given individual's data being scattered across various services. The concept was if a user gives us their name and phone number, is that enough? If so, we aren't interested in anything else. It in theory should benefit the individual and the UX of our product.

Do you have an issue with the above? Would you like me to document this intention somewhere? How can I better communicate how we use Cognito?

> VISA and Plaid

It should be fairly obvious, but our early-stage Bitcoin startup has _zero_ relation to VISA making a multi-billion dollar acquisition. I have clarified [publicly](https://www.youtube.com/watch?v=BKxrLCzMLLo) on many occasions that our relationship with VISA is through their [Fast Track](https://usa.visa.com/partner-with-us/info-for-partners/info-for-fintechs.html) program and our relationship with Plaid is simply signing up, creating an account, and integrating with their API.

A user has a relationship directly with Plaid. This is outlined in their [documentation
](https://plaid.com/docs/). When a user adds a payment method through Plaid (regardless of the service), they are shown the following screen:

![Image from iOS](https://user-images.githubusercontent.com/4040039/91595101-9a01e300-e928-11ea-99e1-d88fd8e63bf9.png)

The user, therefore, agrees to Plaid's terms and privacy policy before adding a payment method. Plaid's privacy policies can be found [here](https://plaid.com/legal/#Privacy-Policy).

Plaid is the number one service in the US to allow deposits from a consumer's bank account. Services such as CashApp, Venmo, Robinhood, and Coinbase use Plaid. I stated publicly (on your podcast @Enegnei) that our effort with Plaid was based on advice from others and the quickest way to accomplish an MVP. We allow users to deposit funds via a debit card currently and plan on allowing other ways for users to deposit funds.

I understand the concern that Plaid is not mentioned in our privacy policy and have reached out to our counsel with an ask to have that updated.

> At this time, contrary to my initial determination, I do not believe that applying this term to Strike would be accurate

How would you define it then? Is it fair to make a distinction between full KYC and what we provide? So far, LIGHT has worked as a way to communicate the feature, however, I am open to suggestions of course.

### Summary

In summary, Strike is a regulated, proprietary service. It does not market itself otherwise and does not market itself to be for those that don't want to share any personal information. It requires an individual's personal information and the ability for that given individual to deposit fiat currency. In order to support the service, third party relationships are required. If one is not comfortable with companies like VISA, Plaid, and so on, that is fair and entirely your decision.

After reviewing your piece @Enegnei it's unclear to me what exactly we need to improve on. Yes, we are regulated and require personal information by law. Yes, we require depositing fiat currency which in turn requires some form of linkage to a payment method. I believe all of our efforts have been public, transparent, and honest. I've publicly stated this project is in BETA and we have been iterating based on feedback and concerns and intend to continue doing so.

**Comment by Enegnei** [comment]
**TLDR;**
Hi @JimmyMow. I really appreciate that you took the time to not only read this privacy-focused review of Strike, but are making / have made changes based on my concerns. If you feel offended in any way by the review, I apologise, I did not mean any disrespect. This document was a collection of my notes, and I was doing my best to produce an honest review.

Don't at all feel obligated to read further, but I wrote up a longer detailed response below to some things you said in the issue and on Twitter, in case it clarifies anything from my end.

--

> I agree this can be misleading. The intention is data we have is never shared and one can spend money to friends and on services through us as a proxy of sorts. However, with Cognito and Plaid, I can understand the concern with displaying `Private` without further clarification. This has been updated to `Fast` on our [website](https://beta.strike.me/).

Great! :thumbsup:

> > They also reserve the right to "block access to the Services from certain IP addresses and unique device identifiers."
> 
> Of course. This is not unique to our service. If we need to block service for a user for regulatory, compliance, or malicious behavior we need to ensure we have that right.

As a privacy-focused review, my goal was to thoroughly highlight instances of data collection, sharing, or analysis, and any inconsistencies. I am looking at "where data flows" (see folder name). So I pointed out that device identifiers are something you collect and may use. I am not necessarily making an argument about whether those actions are right / wrong, should / shouldn't be done. I understand that many, many services also block access based on IP or device identifiers, and that there can be many good reasons for doing so too.

> Reserving the right to change limits is based on our ongoing dialog with regulators. Currently, a LIGHT user has the following limits:
>     * $100 per payment
>     * $2,000 lifetime
> 
> I work with regulators to raise these limits and eventually remove them. This was [discussed publicly](https://www.youtube.com/watch?v=BKxrLCzMLLo) (on your podcast @Enegnei)

Yes, I recall, and I'm glad you're doing so. To clarify, the reason I quoted that section is because it says that you may change the limits for a particular user based on "risk assessment." Obviously, that risk assessment is based on data points that you've collected and analysed (manually or automatically) about the user.

> > However they also use "pixel tags / web beacons," and engage in other automated collection activities:
> 
> Technology such as Cookies is required for such features as persisting a user's login. Web beacons or pixel tags are the politically correct, lawyer-speak, way of acknowledging we use analytical tools. The definition is defined in the Privacy Policy. This is standard across all web and mobile applications.

Yes, I understand. Again, I highlighted it because it is an example of actual or potential data collection, sharing, or analysis, and I am aware of various ways in which both cookies and [web beacons](https://iapp.org/resources/article/web-beacon/) produce metadata trails about user behavior. I am not making an argument that this is unique or malicious on Strike's part.

> > Cognito
> 
> Yes, it is true that we use Cognito to do a wider search based on the information provided to us. [I clarified publicly](https://www.youtube.com/watch?v=BKxrLCzMLLo) (on your podcast @Enegnei) that this "LIGHT KYC" effort was designed to prevent users from sharing information with a third party (Strike) if they don't have to. One of the many dangers KYC authors is a given individual's data being scattered across various services. The concept was if a user gives us their name and phone number, is that enough? If so, we aren't interested in anything else. It in theory should benefit the individual and the UX of our product.
> 
> Do you have an issue with the above? Would you like me to document this intention somewhere? How can I better communicate how we use Cognito?

I will refer back to the feature description (that you have since fixed): "_Strike is your payment proxy to the world. Your information is kept private and never shared_." I highlighted your use of Cognito because it contradicts the statement on Strike's website that you "never share" user information. But information must be shared with Cognito (albeit minimal) in order to use the app. In order to function at all, Cognito relies on "a given individual's data being scattered across various services" already, elsewhere, to check whether a user of yours is "real." Strike is not Cognito, but there is a data flow between the two.

I understand that your / Strike's intention is to minimise the user data that you collect, despite some unfortunately unavoidable requirements given the nature of your business. That is a great intention. But you are still relying on _others__ collecting, sharing, and accessing personal information, as that is Cognito's business model, and so I am highlighting that fact.

> > VISA and Plaid
> 
> It should be fairly obvious, but our early-stage Bitcoin startup has _zero_ relation to VISA making a multi-billion dollar acquisition. I have clarified [publicly](https://www.youtube.com/watch?v=BKxrLCzMLLo) on many occasions that our relationship with VISA is through their [Fast Track](https://usa.visa.com/partner-with-us/info-for-partners/info-for-fintechs.html) program and our relationship with Plaid is simply signing up, creating an account, and integrating with their API.

I did not say anywhere that you were involved in Visa's Plaid acquisition, and I remember you saying on the podcast that you weren't even aware of that deal when you decided to use Plaid. I included that background information in my review to give the full picture of what is currently happening surrounding Plaid with the class action lawsuits; the acquisition deal is directly cited in those legal documents.

> A user has a relationship directly with Plaid. This is outlined in their [documentation ](https://plaid.com/docs/). When a user adds a payment method through Plaid (regardless of the service), they are shown the following screen:
> 
> ![Image from iOS](https://user-images.githubusercontent.com/4040039/91595101-9a01e300-e928-11ea-99e1-d88fd8e63bf9.png)
> 
> The user, therefore, agrees to Plaid's terms and privacy policy before adding a payment method. Plaid's privacy policies can be found [here](https://plaid.com/legal/#Privacy-Policy).

I'm glad to see that Plaid is explicitly mentioned in the Strike user flow. I already quoted what you said during the podcast, about limiting the permissions they are given. Prior to that interview, I had no idea what _your_ Plaid integration collected about Strike users, as I could not find this documented anywhere. What _they_ claim to _generally_ collect is much more invasive than what you described from a privacy perspective.

If you haven't looked through the class action lawsuits against Plaid, I would recommend doing so, at least to better understand why I and others think it's important that they be scrutinized. As the lawsuits explain, when it comes to financial apps, there is a greater expectation that these services are clear about data privacy policies, and where data is being shared. Not only do Plaid and their clients have a history of not disclosing their involvement clearly enough, but Plaid's founders have repeatedly boasted that consumers don't know about them.

> I understand the concern that Plaid is not mentioned in our privacy policy and have reached out to our counsel with an ask to have that updated.

Great! :thumbsup:

> > At this time, contrary to my initial determination, I do not believe that applying this term to Strike would be accurate
> 
> How would you define it then? Is it fair to make a distinction between full KYC and what we provide? So far, LIGHT has worked as a way to communicate the feature, however, I am open to suggestions of course.

The reason I was even prompted to do this review is because I myself initially described Strike as a "KYC-lite" service to others, [leading up to](https://twitter.com/J9Roem/status/1272592841743781892) and [shortly after launch](https://twitter.com/J9Roem/status/1279463380642926592). However, post-launch, I received and read *many* complaints about how that description was inaccurate. To be clear, I define "KYC-lite" as the collection of minimal personally identifying information and especially information that could be spoofed.

For example, there are a lot of light wallets which ask for: your name, phone number, and / or email address. These are KYC-lite because while many users will provide their actual legal name, email address, and personal phone number, a privacy conscious user may give a pseudonym, use Tor / VPN, and a virtual number, and the app will still function for them. I am not the only one who defines KYC-lite this way, and I think the response I got to using it (I believe) incorrectly in the case of Strike is a testament to that current consensus.

I agree that there is still a difference between what Strike does and what collect-it-all-by-default services like Coinbase do. I just don't think KYC-lite is the right term, given that Strikes users will be required to provide much more than just their name and phone number (as the Strike terms / privacy policy spell out) when they exceed the limits or are deemed "risky." Maybe the term should be expanded, but just be aware that there a lot of people who currently define it much more narrowly.

> After reviewing your piece @Enegnei it's unclear to me what exactly we need to improve on. Yes, we are regulated and require personal information by law. Yes, we require depositing fiat currency which in turn requires some form of linkage to a payment method. I believe all of our efforts have been public, transparent, and honest. I've publicly stated this project is in BETA and we have been iterating based on feedback and concerns and intend to continue doing so.

The points for improvement were summarised in the [Conclusion](https://github.com/Enegnei/This-Month-In-Bitcoin-Privacy/blob/master/_posts/where-data-flows/Zap-Solutions-Strike.md#conclusion). And as of opening this issue, you've indicated that Strike has / will satisfy the two changes I was looking for, namely addressing some misleading language and including Plaid in your privacy policy.

**Addendum**

I did not see your [tweets](https://twitter.com/JackMallers/status/1299401654148759553) to me until I had typed most of this response, so I'll address that here. I did raise both of these issues (misleading language re: privacy and Plaid being absent from your privacy policy) directly with you during the podcast interview last month, in the latter half. But I also understood that it was a very busy time for you / Strike, and that Strike is still in BETA, so I wasn't expecting anything to change overnight. I made sure to include the clarifications you did make during the podcast in this document. The context in which I shared in on Twitter, for the first time, yesterday, was due to Matt Odell having a similar experience with Casa (where he raised privacy issues with them publicly and directly over two months ago).

If you look at the revision history for that document, you will see that I wrote most of it around the time of our interview - because these are notes that I had collected during prep for the podcast. [Many people were asking me to research on Cognito and Strike](https://twitter.com/J9Roem/status/1279487830914404352), so I made it public (albeit in a sub-folder of my repo) for them to read. A lot of my writing / journalism is organised and published this way, not out of the ordinary. If at the two-month mark (coming soon) I still hadn't seen changes with the Strike website, I was planning to DM you a reminder and link to this document as backup.

Again I apologise if I offended you in any way with this review, and I really appreciate that you've made changes nonetheless in response to it! :relaxed:
